app/Database/seeds/settings.yml

```diff
@@ -235,6 +235,27 @@
   options: ''
   type: number
   description: 'How much the load factor can vary per-flight'
+- key: flights.different_cargo_load_factor
+  name: 'Different Cargo Load Factor'
+  group: flights
+  value: false
+  options: ''
+  type: boolean
+  description: 'When enabled, v7 will use a different load factor for cargo and passengers fares'
```

Comment:
`When enabled these values will be used for cargo fares` maybe :)

Reply:
Looks good